### PR TITLE
[CoreBundle] allow store-values to be reset and inherit again

### PR DESCRIPTION
**Pull request by dpfaffenbauer**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | yes
| BC breaks?    | no
| Deprecations? |no
| Fixed tickets | #1243 

Found a way

https://recordit.co/Vcmt86YIkb

### CODE REVIEW

File: src/CoreShop/Bundle/CoreBundle/Resources/translations/admin.en.yml [hunk 1]
```diff
@@ -213,3 +213,5 @@ coreshop_solidify_variant_unit_definition_data_error_no_unit_definitions: 'Produ
 coreshop_solidify_variant_unit_definition_data_strategy_only_unit_definitions: 'Are you sure to apply defined unit definitions to each variant? This operation cannot be undone! After you have executed this command, each variant must be managed separately. Also, all depending variants will be closed afterwards, unsaved changes will be lost. Variants with custom unit definitions will be skipped.'
 coreshop_solidify_variant_unit_definition_data_strategy_only_unit_definitions_and_qpr: 'Are you sure to apply defined unit definitions AND quantity price rules to each variant? This operation cannot be undone! After you have executed this command, each variant must be managed separately. Also, all depending variants will be closed afterwards, unsaved changes will be lost. Variants with custom unit definitions and quantity price rules will be skipped.'
 coreshop_solidify_variant_unit_definition_data_succesfully_applied: 'All variants has been successfully solidified. Affected Variants: {0}.'
+coreshop_restore_inheritance: 'Restore inheritance'
+coreshop_restore_inheritance_message: 'Are you sure to delete this Store Values form this Product and let it inherit from its parent again?'
```
Comment (jdreesen) [inline src/CoreShop/Bundle/CoreBundle/Resources/translations/admin.en.yml]:
```suggestion
coreshop_restore_inheritance_message: 'Are you sure to delete the Store Values form this Product and let it inherit from its parent again?'
```

File: src/CoreShop/Bundle/CoreBundle/Resources/translations/admin.de.yml [hunk 2]
```diff
@@ -214,3 +214,5 @@ coreshop_solidify_variant_unit_definition_data_error_no_unit_definitions: 'Das P
...
 coreshop_solidify_variant_unit_definition_data_strategy_only_unit_definitions_and_qpr: 'Sind Sie sicher, dass Sie die Einheiten-Definitionen UND Stückmengen-Preisregeln auf jede Variante überragen wollen? Dieser Vorgang kann nicht rückgängig gemacht werden! Nachdem Sie diesen Befehl ausgeführt haben, muss jede Variante separat verwaltet werden. Zudem werden alle Varianten, welche dem Produkt angehören, geschlossen. Nicht gespeicherte Änderungen gehen dabei verloren. Varianten, welche bereits Einheiten-Definitionen oder Stückmengen-Preisregeln besitzen, werden übersprungen.'
 coreshop_solidify_variant_unit_definition_data_succesfully_applied: 'Alle Daten wurden erfolgreich auf die Varianten übertragen. Verarbeitete Datensätze: {0}.'
+coreshop_restore_inheritance: 'Vererbung wiederherstellen'
+coreshop_restore_inheritance_message: 'Sind Sie sicher das Sie die Store Eigenschaften des Produktes löschen und vom Mutter Element erben lassen wollen?'
```
Comment (jdreesen) [inline src/CoreShop/Bundle/CoreBundle/Resources/translations/admin.de.yml]:
```suggestion
coreshop_restore_inheritance_message: 'Sind Sie sicher das Sie die Store-Eigenschaften des Produktes löschen und vom Eltern-Element erben lassen wollen?'
```

File: src/CoreShop/Bundle/CoreBundle/Resources/translations/admin.en.yml (hunk 1 above)
Comment (dpfaffenbauer) [reply src/CoreShop/Bundle/CoreBundle/Resources/translations/admin.en.yml]: 👍 , but I was about to merge this PR, you can change the translation later on crowdin.com

File: src/CoreShop/Bundle/CoreBundle/Resources/translations/admin.de.yml (hunk 2 above)
Comment (dpfaffenbauer) [reply src/CoreShop/Bundle/CoreBundle/Resources/translations/admin.de.yml]: 👍 , but I was about to merge this PR, you can change the translation later on crowdin.com

File: src/CoreShop/Bundle/CoreBundle/Resources/translations/admin.en.yml (hunk 1 above)
Comment (jdreesen) [reply src/CoreShop/Bundle/CoreBundle/Resources/translations/admin.en.yml]: Is it possible to update the English language on crowdin, too?

Comment (dpfaffenbauer) [reply src/CoreShop/Bundle/CoreBundle/Resources/translations/admin.en.yml]: yes

**Pull request merged**